scripts/debug/stage1_install_busybox.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+
+ARGC=$#
+
+if [ "${ARGC}" -lt 1 ]; then
+    echo "Usage: install_busybox.sh UUID"
+    exit 1
+fi
+
+UUID="${1}"
+
+BUSYBOX=${BUSYBOX_BINARY:-$(which busybox 2> /dev/null)}
+
+if [ -z "${BUSYBOX}" ]; then
```

Comment:
-x instead of -z?
echo "error: busybox is not executable: Install it or set BUSYBOX_BINARY env variable"